form_subscribe.html

```diff
@@ -1,23 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```

Comment:
We think this is still the same as the commit message, you've done different thing from what you've described. Cheers!